code/modules/vending/vending.dm

```diff
@@ -248,6 +248,20 @@ ADMIN_INTERACT_PROCS(/obj/machinery/vending, proc/throw_item, proc/admin_command
 		else
 			return src.loc
 
+	HELP_MESSAGE_OVERRIDE({""})
```

Comment:
I think that's needed as part of dynamic help messages https://github.com/goonstation/goonstation/blob/8710cf3a0f7bde2d047d57843ecf368557cd37a6/.github/HELP_MESSAGES.md#code . When I remove it, the help messages work, but always show up when examining the object instead of only on the help right click action.